q2_diversity_lib/_alpha/_method.py

```diff
@@ -32,3 +34,44 @@ def faith_pd(table: biom.Table, phylogeny: skbio.TreeNode) -> pd.Series:
 
     result.name = 'faith_pd'
     return result
+
+
+def observed_features(table: biom.Table) -> pd.Series:
+    if table.is_empty():
+        raise ValueError("The provided table object is empty")
+    presence_absence_table = table.pa()
+    counts = presence_absence_table.matrix_data.toarray().astype(int).T
+    sample_ids = presence_absence_table.ids(axis='sample')
+    result = skbio.diversity.alpha_diversity(metric='observed_otus',
+                                             counts=counts, ids=sample_ids)
+    result.name = 'observed_features'
+    return result
+
+
+def pielou_evenness(table: biom.Table, drop_nans: bool = False) -> pd.Series:
+    if table.is_empty():
+        raise ValueError("The provided table object is empty")
+    counts = table.matrix_data.toarray().T
+    sample_ids = table.ids(axis='sample')
+    if drop_nans:
+        counts, sample_ids = _drop_nans(
+                counts, sample_ids, minimum_nonzero_elements=2)
+
+    result = skbio.diversity.alpha_diversity(metric='pielou_e', counts=counts,
+                                             ids=sample_ids)
+    result.name = 'pielou_e'
```

Comment:
Since we are already tweaking metric names a bit (see `observed_features` above) - what if we expanded this to `pielou_evenness`? I suspect that a new user might not know what `e` means here (entropy? evenness? encumbrance?).